runtime/runtime/src/lib.rs

```diff
@@ -1453,7 +1453,8 @@ impl Runtime {
         state_update.commit(StateChangeCause::Migration);
     }
 
-    /// It's okay to use unsafe math here, because this method should only be called on the trusted
+    /// Computes the expected storage per account for a given set of StateRecord(s).
+    /// Note: It's okay to use unsafe math here, because this method should only be called on the trusted
     /// state records (e.g. at launch from genesis)
```

Comment:
The note should probably be moved to the `StorageComputer`

Reply:
done.